### ISSUE TITLE
Last played within x time

### ISSUE THREAD

**Issue by Seishirou101** [blacktwin/JBOPS#253]
May be improvement to https://github.com/blacktwin/JBOPS/blob/master/notify/find_unwatched_notify.py

I would like to have the script to also check if a certain show/movie has been watched in x time. Basically I would like a notification something like, "Game of Thrones S00E99 has not been watched for over 6 months".

Currently, I usually go into tautulli and sort by last played and sift through the dates that way but would love an automated way to pull a list from x time ago. 

thank you very much

**Comment by blacktwin** [comment]
There are a couple scripts that find unwatched items but I've consolidated them into one, https://github.com/blacktwin/JBOPS/blob/master/utility/media_manager.py. This is also bound to "whats been added since X and not watched" instead of your request of "what hasn't been watched in X time". If I do any work, it'll be in the media_manager.py script. You aren't the only one to ask for this, it makes sense, and sounds like a good idea. I would also need to add the notification piece to media_manager.py.

**Comment by blacktwin** [comment]
@Seishirou101 Please try out PR #270 , I'll be testing as well.
```
media_manager.py --select lastPlayed --action show --libraries Movies --date 2020-01-01
Checking library: 'Movies' watch statuses...
The following items were last played before 2020-01-01
	Movie1 added 2015-05-13 and last played 2019-12-08	Size: 4.1GiB
		File: F:\media\Movies\Movie1\Movie1.mkv
	Movie2 added 2017-12-30 and last played 2019-12-08	Size: 5.4GiB
		File: F:\media\Movies\Movie2\Movie2.mkv
```

**Comment by Seishirou101** [comment]
Thank you so much! I was playing around with it and so far, .  One of my libraries also kicked out with this error as well. Here is the output following your example above with my library labeled "TV Shows"
`python3 media_manager.py --select lastPlayed --action show --libraries 'TV Shows' --date 2020-01-01
Checking library: 'TV Shows' watch statuses...
The following items were last played before 2020-01-01
	Teen Titans added 2019-01-16 and last played 2019-12-28	Size: 54.6GiB
		File: /data/tvshows/TV Shows/Teen Titans
	The Dragon Prince added 2019-12-19 and last played 2019-12-27	Size: 16.8GiB
		File: /data/tvshows/TV Shows/The Dragon Prince
	Scooby-Doo! Mystery Incorporated added 2019-12-15 and last played 2019-12-25	Size: 20.4GiB
		File: /data/tvshows/TV Shows/Scooby-Doo! Mystery Incorporated
	Voltron: Legendary Defender added 2018-12-16 and last played 2019-12-23	Size: 65.5GiB
		File: /data/tvshows/TV Shows/Voltron- Legendary Defender
	Band of Brothers added 2019-09-22 and last played 2019-10-19	Size: 5.8GiB
		File: /data/tvshows/TV Shows/Band of Brothers
	Marvel's Cloak & Dagger added 2019-02-19 and last played 2019-10-16	Size: 14.6GiB
		File: /data/tvshows/TV Shows/Marvel's Cloak & Dagger
	Star Wars Rebels added 2019-01-06 and last played 2019-10-04	Size: 82.7GiB
		File: /data/TV Shows_backup/Star Wars Rebels
	Young & Hungry added 2019-07-09 and last played 2019-09-11	Size: 16.5GiB
		File: /data/tvshows/TV Shows/Young & Hungry
	The Fresh Prince of Bel-Air added 2018-12-14 and last played 2019-04-28	Size: 119.3GiB
		File: /data/TV Shows_backup/The Fresh Prince of Bel-Air
	Atlanta added 2019-01-14 and last played 2019-03-31	Size: 52.3GiB
		File: /data/tvshows/TV Shows/Atlanta
Traceback (most recent call last):
  File "media_manager.py", line 675, in <module>
    added_at = datetime.datetime.utcfromtimestamp(float(item.added_at)).strftime("%Y-%m-%d")
TypeError: float() argument must be a string or a number, not 'NoneType'
`

**Comment by blacktwin** [comment]
That item doesn't have a `added_at` date. It would kill the script. When every the action of `show` is used the steps are pretty much the same for each selector. I should make it it's own function and wrap it in a `try: except:`. Thanks!

**Comment by blacktwin** [comment]
This issue should now be resolved via #270
